client/src/Features/PublishAssignment/PublishSuccessMessageBar.tsx

```diff
@@ -60,11 +64,15 @@ const PublishSuccessMessageBarInner = ({
 
 const publishSuccessMessageBarStyles = ({ theme: _theme }: IThemeOnlyProps): PublishSuccessMessageBarStyles => ({
```

Comment:
please see if this works well (and corresponding changes in the following lines).. just trying to make it consistent with the other files
```suggestion
const publishSuccessMessageBarStyles = ({ theme }: IThemeOnlyProps): PublishSuccessMessageBarStyles => ({
```

Reply:
Yeah, I agree, although I didn't change it to avoid any unnecessary modifications, but let's make it consistent!

Reply:
thanks for that!